library/src/services/api/ParsedApiSpecInterface.ts

```diff
@@ -86,6 +85,7 @@ export interface ScalarProperty {
   required: boolean;
   pattern?: RegExp;
   helpText?: string | React.ReactElement<any>;
+  downloadable?: boolean;
```

Comment:
This will require the developer to activate file download feature on every single column. We could just auto-detect it and show the download button

Reply:
That is true, and this decision is taken because sometimes we want to show the download icon in a specific column of a downloadable file and maintain the current behavior in other fields that can contain downloadable items but not wants to be shown in the column